charts/opentelemetry-collector/templates/_pod.tpl

```diff
@@ -128,4 +128,5 @@ affinity:
 tolerations:
   {{- toYaml . | nindent 2 }}
 {{- end }}
+hostNetwork: {{ .Values.hostNetwork }}
```

Comment:
Since hostNetwork is added in the _pod.tpl, which is used in both the daemonset and deployment yamls, is there a risk that if the chart is installed with both agent and standalone collectors that the agent and deployment will experience port conflict on any nodes they are both running?